app/controllers/users_controller.rb

```diff
@@ -12,6 +12,7 @@ def show
   def create
     @user = User.new(user_params)
     if @user.save
+      AppMailer.send_welcome_email(@user).deliver_now
```

Comment:
`deliver_now` - I wanted to comment this is working since Rails 4.2, but I checked it and you have upgraded it!